### PR TITLE
feat: update audio-related plugins

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add or update audio-related plugins.

## How Has This Been Tested?

- Installation by release version apm is possible.

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: v3/list.json [hunk 1]
```diff
@@ -10,7 +10,11 @@
   "packages": [
     {
       "path": "packages.json",
-      "modified": "2022-06-18T05:48:00+09:00"
+      "modified": "2022-06-18T12:48:00+09:00"
+    },
+    {
+      "path": "package-sets.json",
+      "modified": "2022-06-16T05:48:00+09:00"
```
Comment (hal-shu-sato) [inline v3/list.json:17]: apm can catch this not-found error but the error is not good.

**Review by hal-shu-sato — COMMENTED** [review]
I have one point to review.

**Comment by mitosagi** [comment]
I have corrected it.

**Review by hal-shu-sato — APPROVED** [review]
Good addition.

**Pull request merged**